projectatomic/ai-studio
# feat: Playgrounds navigation

### What does this PR do?

Adds a Playground page, and the possibility to create a new playground with an optional name, and a given model.

When a new playground is created and no model service is running for its model, the model server is created and started. If the model server exists but is stopped, it is started.


Out of scope for this PR:
- the playground details page only displays the playground id
- the PlaygroungManagerV2 API needs to be adapted to accept playgroundId as input, and not accept info already known by the playground (modelId)
- the user is not redirected yet to the playground page (as it is empty for now)

### Screenshot / video of UI


https://github.com/projectatomic/ai-studio/assets/9973512/7748094d-d755-4ae7-8696-cda4cf213fa0

No models are downloaded yet:

![no-model](https://github.com/projectatomic/ai-studio/assets/9973512/e4a29ebd-6629-4682-9c4f-d93acfa57197)

More models could be downloaded:

![some-models](https://github.com/projectatomic/ai-studio/assets/9973512/9fe5ede7-44d9-4a6e-9327-b98a5fb9893e)

All models are downloaded:

![all-models](https://github.com/projectatomic/ai-studio/assets/9973512/a35d95c0-b006-42d4-8786-fc19474eac10)



### What issues does this PR fix or reference?

Part of #525 

### How to test this PR?

Go to the Playground page and use "New Playground" button to create new playgrounds.

Creating new playgrounds should start "Services" for the model chosen for the playground, if not already running

Depending on the models downloaded or not, different messages should appear (see screenhost above)
